packages/tupaia-web/src/pages/VerifyEmailResend.tsx

```diff
@@ -0,0 +1,63 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import Typography from '@material-ui/core/Typography';
+import { AuthModalBody, AuthModalButton, TextField } from '../components';
+import { FORM_FIELD_VALIDATION } from '../constants';
+import { SubmitHandler, useForm } from 'react-hook-form';
+import { useResendVerificationEmail } from '../api/mutations';
+
+const ModalBody = styled(AuthModalBody)`
+  width: 53rem;
+`;
+
+const CheckEmailMessage = styled.p`
+  text-align: center;
+  padding: 0 15px;
+`;
+
+const StyledForm = styled.form`
+  margin-top: 1rem;
+  width: 21rem;
+  max-width: 100%;
```

Comment:
I wonder if we can do the width here as a percentage as well, since the modal body has a set width?

Reply:
Same as before this is intentionally set to a width that is less than the outer width to match the design as an alternative to adding padding as I find that it's a nice way to make all the widths work nicely responsively.